osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -124,8 +134,11 @@ private double computeAimValue(ScoreInfo score, OsuDifficultyAttributes attribut
 
             if (attributes.SliderCount > 0)
             {
-                double estimateSliderEndsDropped = Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders);
-                double sliderNerfFactor = (1 - attributes.SliderFactor) * Math.Pow(1 - estimateSliderEndsDropped / estimateDifficultSliders, 3) + attributes.SliderFactor;
+                double estimateSliderEndsDropped;
+                estimateSliderEndsDropped = usingClassicSliderAccuracy ? Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders) : Math.Min(countSliderEndsDropped + countLargeTickMiss, estimateDifficultSliders);
```

Comment:
This is a pain to read, please use normal `if-else`

Reply:
I agree but making it not a ternary causes it to fail the code quality test. 
![image](https://github.com/user-attachments/assets/3332eecb-4cfa-4ac0-ade9-a635a3ed5e6d)
Is passing this test not a requirement?

Reply:
Oh, that's kinda stupid. I guess another option would be at least adding some linebreaks instead of having it as one giant line

Reply:
It is, and I wouldn't be against relaxing that rule. How do @peppy @bdach feel about relaxing the convert-to-ternary warning on if-statements? I find that its relative use is pretty subjective.

Reply:
Can be changed to a hint if you want.

But usually we just use newlines to make this as-legibile-as-if?

```csharp
                estimateSliderEndsDropped = usingClassicSliderAccuracy
                    ? Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders)
                    : Math.Min(countSliderEndsDropped + countLargeTickMiss, estimateDifficultSliders);
```

Either or, just feels weird to change the rule after being okay for this long.